shared/utils/generate-from-templates.py

```diff
@@ -118,38 +119,42 @@ def _get_csv_dir(self):
         return self.csv_dirs[self.current_oval]
 
     def _get_csv_list(self):
-        dir = self._get_csv_dir()
+        dir_ = self._get_csv_dir()
 
         csvs = []
 
         try:
-            files = os.listdir(dir)
+            files = os.listdir(dir_)
         except OSError:
             return []
 
-        for file in files:
+        for file_ in files:
             # skip non csv files
-            if not file.endswith(".csv"):
+            if not file_.endswith(".csv"):
                 continue
 
             # skip empty files
-            filepath = os.path.join(dir, file)
+            filepath = os.path.join(dir_, file_)
             if os.stat(filepath).st_size == 0:
                 continue
 
-            csvs.append(file)
+            csvs.append(file_)
+
         return csvs
 
     def _get_script_for_csv(self, csv_filename):
         try:
             script_name = self.script_dict[csv_filename]
             full_path = os.path.join(self.shared_templates_dir, script_name)
             return full_path
+
         except KeyError:
-            sys.stderr.write("Cannot find associated build script for {0}\n".format(csv_filename))
+            sys.stderr.write(
+                "Cannot find associated build script for {0}\n"
+                .format(csv_filename)
+            )
             sys.exit(1)
 
-
     def _parent_output_dir(self):
         return os.path.join(self.output_dir, "output")
```

Comment:
I recall discussion about appending "output" to the specified --output directory name.  If we remove this line, then you might be able to remove the "rm -rf templates/output" from the Makefile changes above.